instrumentation/jms/src/test/java/brave/jms/JmsExtension.java

```diff
@@ -100,12 +100,12 @@ abstract void setReadOnlyProperties(Message message, boolean readOnlyProperties)
 
   @Override public void afterEach(ExtensionContext context) throws Exception {
     try {
-      session.close();
-      connection.close();
-      topicSession.close();
-      topicConnection.close();
-      queueSession.close();
-      queueConnection.close();
+      if (session != null) session.close();
```

Comment:
when the first flake happened, it was obscured by a NPE, so I fixed in both places